src/index.js

```diff
@@ -5,6 +5,7 @@ const { exec } = require('child_process');
 
 // External dependencies
 const Hapi = require('hapi');
+const HAPIWebSocket = require('hapi-plugin-websocket');
```

Comment:
Cqan you change it for `hapiWebSocket` ?

Reply:
Excellent work btw :)

Reply:
I've updated the what you asked, and thank you.